Formula/nginx.rb

```diff
@@ -3,8 +3,8 @@ class Nginx < Formula
   homepage "https://nginx.org/"
   # Use "mainline" releases only (odd minor version number), not "stable"
   # See https://www.nginx.com/blog/nginx-1-12-1-13-released/ for why
-  url "https://nginx.org/download/nginx-1.17.3.tar.gz"
-  sha256 "3b84fe1c2cf9ca22fde370e486a9ab16b6427df1b6ea62cdb61978c9f34d0f3c"
+  url "https://nginx.org/download/nginx-1.17.4.tar.gz"
+  sha256 "62854b365e66670ef4f1f8cc79124f914551444da974207cd5fe22d85710e555"
   revision 1
```

Comment:
`revision` here should be removed.
`passenger` formula needs a revision bump (in a separate commit)